app/src/main/kotlin/io/homeassistant/companion/android/util/compose/media/player/HAMediaPlayer.kt

```diff
@@ -0,0 +1,346 @@
+    }
+}
+
+@Composable
+private fun BoxScope.ShowControlsButton(action: () -> Unit) {
+    Spacer(
+        modifier = Modifier
+            .fillMaxSize()
+            .clickable(
+                interactionSource = remember { MutableInteractionSource() },
+                // Prevent ripple on tap
+                indication = null,
+                onClick = action,
+            ),
+    )
+}
+
+@Composable
+private fun BottomControls(
+    player: Player,
+    isFullScreen: Boolean,
+    onClickFullscreen: () -> Unit,
+    modifier: Modifier,
+) {
+    Row(
+        modifier = modifier
+            .height(BottomControlsHeight)
+            .fillMaxWidth()
+            .background(
+                Brush.verticalGradient(
+                    colors = listOf(ControlBackgroundColorStart, ControlBackgroundColorEnd),
+                ),
+            ),
+    ) {
+        TimeText(player)
+        Spacer(modifier = Modifier.weight(1f))
+        MuteButton(player)
+        FullscreenButton(isFullScreen, onClickFullscreen)
+    }
+}
+
+@Composable
+private fun RowScope.FullscreenButton(isFullScreen: Boolean, onClickFullscreen: () -> Unit) {
+    IconButton(
+        onClick = onClickFullscreen,
+        modifier = Modifier
+            .align(Alignment.CenterVertically)
+            .size(BottomControlsHeight),
+    ) {
+        Icon(
+            if (isFullScreen) Icons.Default.FullscreenExit else Icons.Default.Fullscreen,
+            contentDescription = stringResource(R.string.fullscreen),
+            tint = Color.White,
+            modifier = Modifier.size(BottomControlButtonSize),
+        )
+    }
+}
+
+@OptIn(UnstableApi::class)
+@Composable
+private fun RowScope.MuteButton(player: Player) {
+    val muteState = rememberMuteUnmuteButtonState(player)
+    if (muteState.isEnabled) {
+        IconButton(
+            modifier = Modifier
+                .align(Alignment.CenterVertically)
+                .size(BottomControlsHeight),
+            onClick = muteState::onClick,
+        ) {
+            val icon =
+                if (muteState.showMute) Icons.AutoMirrored.Filled.VolumeOff else Icons.AutoMirrored.Filled.VolumeUp
+            Icon(
+                icon,
+                contentDescription = stringResource(R.string.mute_unmute),
+                modifier = Modifier.size(BottomControlButtonSize),
+                tint = Color.White,
+            )
+        }
+    }
+}
+
+@Composable
+private fun RowScope.TimeText(player: Player) {
+    var currentPosition by remember { mutableStateOf(player.currentPositionDuration()) }
+
+    LaunchedEffect(player) {
+        while (true) {
+            currentPosition = player.currentPositionDuration()
+            delay(TIME_TRACKING_UPDATE_DELAY)
+        }
+    }
+
+    Text(
+        currentPosition.toComponents { hours, minutes, seconds, _ ->
+            if (hours > 0L) {
+                String.format(Locale.US, "%02d:%02d:%02d", hours, minutes, seconds)
```

Comment:
I'm not aware of any locale specific notation for a timer, but why not use `Locale.getDefault()`?

Reply:
I'm not to I just fixed a local but I will use default.